Rebracer.xml

```diff
@@ -11,9 +11,14 @@
       <ToolsOptionsSubCategory name="CSharp-Specific">
         <PropertyValue name="AutoComment">1</PropertyValue>
         <PropertyValue name="BringUpOnIdentifier">1</PropertyValue>
+        <PropertyValue name="ClosedFileDiagnostics">1</PropertyValue>
         <PropertyValue name="CodeDefinitionWindow_DocumentationComment_IndentBase">1</PropertyValue>
         <PropertyValue name="CodeDefinitionWindow_DocumentationComment_IndentOffset">2</PropertyValue>
         <PropertyValue name="CodeDefinitionWindow_DocumentationComment_WrapLength">80</PropertyValue>
+        <PropertyValue name="DisplayLineSeparators">0</PropertyValue>
```

Comment:
These are caused by an update to the Visual Studio extension Rebracer that switches code format settings for a solution based on this configuration file. These are just new configuration options that are supported and get added to the file automatically.